### ISSUE TITLE
-Wenum-conversion in drivers/phy/intel/phy-intel-combo.c

### ISSUE THREAD

**Issue by nathanchance** [ClangBuiltLinux/linux#1034]
```
drivers/phy/intel/phy-intel-combo.c:202:34: warning: implicit conversion from enumeration type 'enum intel_phy_mode' to different enumeration type 'enum intel_combo_mode' [-Wenum-conversion]
        enum intel_combo_mode cb_mode = PHY_PCIE_MODE;
                              ~~~~~~~   ^~~~~~~~~~~~~
1 warning generated.
```

Patch submitted: https://lore.kernel.org/lkml/20200523035043.3305846-1-natechancellor@gmail.com/

**Comment by nickdesaulniers** [comment]
accepted: https://git.kernel.org/pub/scm/linux/kernel/git/phy/linux-phy.git/commit/?h=fixes&id=6153224bef8b218ecf9bf541b6154ff72fc4c54b

**Comment by nathanchance** [comment]
Merged into mainline: https://git.kernel.org/linus/6153224bef8b218ecf9bf541b6154ff72fc4c54b
